tests/lib/rss.test.js

```diff
@@ -0,0 +1,21 @@
+/**
+ * RSS Library Test
+ */
+
+const fs = require('fs')
+
+describe('lib.rss', () => {
+  it('should create a feed.xml file', () => {
+    const path = `${__dirname}/../../out/feed.xml`
+    const file = fs.existsSync(path)
```

Comment:
Call the rss function itself with mock data.

Reply:
`main` can export a function, beside running them directly, so that it can be called with mocked fs and then tested with this test.

Reply:
Sure, I'll call the RSS function with mock data, but I don't think using `main` in the `rss` test is very good - defeats the purpose of unit tests. I'll make a main test when I finish refactoring in a new PR.